src/layer/hardswish.cpp

```diff
@@ -24,7 +24,7 @@ HardSwish::HardSwish()
 
 int HardSwish::load_param(const ParamDict& pd)
 {
-    alpha = pd.get(0, 0.2f);
+    alpha = pd.get(0, 1.f / 6);
```

Comment:
This breaks backward compatibility.
Suppose there is one old model file using the default alpha value. You will get different output with or w/o this change.

Reply:
Have reverted it and add some comments instead.